.travis.yml

```diff
@@ -1,5 +1,5 @@
 language: node_js
-script: npm run coverage && cat ./coverage/lcov.info | coveralls
+after_script: npm run coverage && cat ./coverage/lcov.info | coveralls
```

Comment:
It does some of the testing stuff, but not all. `script: npm test` would also run our JSCS and JSHint checks, but since the language is `node_js`, this is used implicitly... I think. We can wait for the build report to verify.